java/example/calypso/android/nfc/src/main/kotlin/org/eclipse/keyple/example/activity/AbstractExampleActivity.kt

```diff
@@ -96,7 +97,7 @@ abstract class AbstractExampleActivity : AppCompatActivity(), NavigationView.OnN
         (reader as ObservableReader).addObserver(this)
 
         // with this protocol settings we activate the nfc for ISO1443_4 protocol
-        reader.configProtocol(ContactlessCardCommonProtocols.ISO_14443_4)
+        reader.configProtocol(AndroidNfcSupportedProtocols.ISO_14443_4)
```

Comment:
Here for a plugin example, we should use the Keyple API : org.eclipse.keyple.core.service.Reader.activateProtocol(String readerProtocolName, String applicationProtocolName)
Not a configProtocol method defined specificly in the example. => in this case the implementation example is too far for the other examples for the other plugins.